presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveSchema.java

```diff
@@ -0,0 +1,285 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.hive;
+
+import com.google.common.collect.ImmutableList;
+import io.prestosql.tempto.AfterTestWithContext;
+import io.prestosql.tempto.BeforeTestWithContext;
+import io.prestosql.tempto.ProductTest;
+import io.prestosql.tempto.assertions.QueryAssert;
+import io.prestosql.tempto.query.QueryExecutionException;
+import io.prestosql.tempto.query.QueryResult;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.Condition;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static com.google.common.base.Strings.nullToEmpty;
+import static com.google.common.base.Verify.verify;
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static io.prestosql.tempto.assertions.QueryAssert.Row.row;
+import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
+import static io.prestosql.tests.TestGroups.STORAGE_FORMATS;
+import static io.prestosql.tests.utils.QueryExecutors.onPresto;
+import static java.util.Objects.requireNonNull;
+
+public class TestHiveSchema
+        extends ProductTest
+{
+    @BeforeTestWithContext
+    public void setUp()
+    {
+        // make sure hive.default schema is not empty
+        onPresto().executeQuery("DROP TABLE IF EXISTS hive.default.test_sys_schema_disabled_table_in_default");
```

Comment:
this is going to be called before each test.  Maybe you could have a variable that would not create this table if it is already created, then `tearDown` could be removed. WDYT?

Reply:
in general -- i agree

it's tiny overhead compared to the number of test queries. i choose to ignore this